extensions/visual-studio-code-snippet-manager/src/data.ts

```diff
@@ -0,0 +1,23 @@
+export const path = `${process.env.HOME}/Library/Application Support/Code/user/snippets/raycast.code-snippets`;
```

Comment:
what about allowing users to choose between VSCode and VSCode Insiders and generating this URL based on the user's selection?

Reply:
I see.
I didn't know VSCode Insiders.

I will do so.